model/map-hot-rod/src/main/java/org/keycloak/models/map/storage/hotRod/IckleQueryMapModelCriteriaBuilder.java

```diff
@@ -175,6 +189,43 @@ private StringBuilder getWhereClauseBuilder() {
         return whereClauseBuilder;
     }
 
+    public static Object sanitize(Object value) {
```

Comment:
This method is unused.

Reply:
Good catch. It should be called from `iLike` method.